libs/parallelism/algorithms/include/hpx/parallel/algorithms/all_any_none.hpp

```diff
@@ -360,13 +338,17 @@ namespace hpx { namespace parallel { inline namespace v1 {
         none_of(ExPolicy&& policy, FwdIter first, FwdIter last, F&& f,
             Proj&& proj = Proj())
     {
-        typedef hpx::traits::is_segmented_iterator<FwdIter> is_segmented;
+        static_assert(hpx::traits::is_forward_iterator<FwdIter>::value,
+            "Required at least forward iterator.");
+
+        typedef hpx::is_sequenced_execution_policy<ExPolicy> is_seq;
```

Comment:
If you can use `using` instead of `typedef`, it would be nice, as we are trying to replace the old `typedef`s in the codebase :)

Reply:
Thank you! You dind't have to go to the trouble of finding every line, just let me know at one spot and leave the searching to me :)